README-VN.md

```
@@ -88,55 +88,62 @@ Astrocelerate có bản commit đầu tiên vào ngày 28 tháng 11 năm 2024. T
+### 2026-04-12
+<img width="1919" height="1031" alt="2026-04-12" src="https://github.com/user-attachments/assets/03028d1b-e055-4e6d-b202-7e734e6bdf75" />
+
+### 2025-12-17
 <img width="1919" height="1031" alt="2025-12-17" src="https://github.com/user-attachments/assets/82a93f52-c7ea-45d6-a5eb-f03534892a36" />
 
-### 20/10/2025
+### 2025-10-20
 <img width="1919" height="1029" alt="2025-10-20" src="https://github.com/user-attachments/assets/6a0309a0-6af9-4c2a-a9a3-4774f811d2b6" />
 
-### 18/10/2025 (sau 2 tháng ngừng phát triển)
+### 2025-10-18 (after 2-month hiatus)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)